### DISCUSSION TITLE
Deepgram responded me with 503 error

### DISCUSSION THREAD

**Discussion by valentinkorzun** [deepgram/community#88] [General help]
Hi!

When I tried to get temporary key:

```js
const response = await DeepgramService.createKey(functions.config().deepgram.project_id, {
    scopes: ['member'],
    time_to_live_in_seconds: 100,
  });
```

Deepgram sometimes responded to me with error 503. Why did it happen?

**Comment by BekahHW** [comment]
Hey @valentinkorzun!

Can you give some more info? Are you using the NodeSDK? When did this occur? Is it currently happening?

**Comment by briancbarrow** [comment]
@valentinkorzun closing this because we haven't heard back from you in a while. Please reach out if you are still running into issues.
